pkg/components/enterprise.go

```diff
@@ -180,5 +185,6 @@ var (
 		ComponentTigeraTypha,
 		ComponentTigeraCNI,
 		ComponentCloudControllers,
+		ComponentCSRInitContainer,
```

Comment:
Won't this make it use the wrong registry?

Reply:
This doesn't have any effect on the registry used. In components/references.go is what controls which registry it comes from. This PR removes it from the switch case that removes it from the Calico registry (where the OS components are) so the default case hits which is the Tigera registry (quay.io or gcr.io for testing/hashreleases).